.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js:
+  - "8"
+cache: yarn
+script: yarn build
+after_success:
+- test $TRAVIS_PULL_REQUEST == "false" && test $TRAVIS_BRANCH == "webpage" && bash
```

Comment:
Shouldn't we run this when both `master` and `website` get updated (if we rebuild the JSON - see my previous comment)?

Reply:
master doesn't have `website` and `build` script. There is nothing we can deploy from master

Reply:
Yep, my idea was (not sure if it's feasible): when there's a new push on either `master` or `website`, we switch to `website`, we run rebuild (data + app) and deploy.
What do you think? 
Thank you very much!

Reply:
Theoretically this is possible, but I do not feel like this the right thing to do.

There are a lot of folders in the root of main project anyway. We can add another one called `website`. Classical mono-repo move. WDYT?

Reply:
Let's do it, since it's going to make the process simpler.
Thank you very much for taking care of this.